.github/workflows/_publish-docs-on-release.yml

```diff
@@ -50,6 +60,13 @@ jobs:
           fi
           python -m pip install . --no-deps
 
+      - name: Start Xvfb
```

Comment:
I have a good feeling that this will fix the API doc building problem for fourigui while using linux

Reply:
yes this has worked: https://github.com/diffpy/diffpy.fourigui/pull/71